main.js

```diff
@@ -313,6 +311,68 @@
       route: "/typography",
       component: <UXWTypography />,
     },
+    {
+      heading: "Component",
+      subheadings: [
+        {
+          subtitle: "Getting Started",
```

Comment:
You don't need a subsection here if you don't need a link on the left.

Reply:
done